lib/nanoc/base/views/post_compile_item_view.rb

```diff
@@ -0,0 +1,11 @@
+module Nanoc
+  class PostCompileItemView < Nanoc::ItemView
+    def created?
+      reps.select { |rep| rep.status == :created }
+    end
+
+    def updated?
+      reps.select { |rep| rep.status == :modified }
```

Comment:
Can you replace `:modified` with `:updated`? This way, it reflects the name of the method.